healvis/sky_model.py

```diff
@@ -175,8 +176,9 @@ def read_hdf5(self, filename, freq_chans=None, shared_memory=False, do_not_overw
                 sky_freqs_full = infile['freqs'][()]
                 freq_chans = np.where(np.in1d(sky_freqs_full, self.freqs))[0]
                 sky_freqs_part = sky_freqs_full[freq_chans]
-                if not np.allclose(self.freqs, sky_freqs_part):
-                    raise ValueError("Currently set frequencies do not match any subset of file's frequencies.")
+                if self.freqs is not None:
```

Comment:
second "if" could be an "and" on the first one

Reply:
Fixed. Thanks!